react/features/video-menu/components/native/RemoteVideoMenu.js

```diff
@@ -63,10 +68,20 @@ type Props = {
      */
     _isOpen: boolean,
 
+    /**
+     * Whether the participant is present in the room or not.
+     */
+    _isParticipantIDAvailable?: boolean,
```

Comment:
why do we need to know this?

Reply:
So, this is for when a remote participant leaves the meeting and we have the context menu details open. If this happens, the context menu closes automatically. If we don't do this, the context menu remains open and the participant details reset.